Pal/src/host/Linux/pal_host.h

```diff
@@ -230,12 +227,14 @@ struct arch_frame {
 #endif
 
 #define PAL_FRAME_IDENTIFIER    (0xdeaddeadbeefbeef)
+#define PAL_LOCK_RECORDS        4
 
 struct pal_frame {
     volatile uint64_t           identifier;
     void *                      func;
     const char *                funcname;
     struct arch_frame           arch;
+    PAL_LOCK *                  lock_records[PAL_LOCK_RECORDS];
```

Comment:
This could use a comment.  I assume the scheme is that you might acquire some locks in an exception handler and need a mechanism to free them?